### PR TITLE
[Remove Stream/Optional usage] LPS-178006 remove_stream_and_optional_in_search-experiences-web

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Dante,

This is for ticket https://issues.liferay.com/browse/LPS-178006.
There is still one optional usage in SXPBlueprintOptionsPortletPreferencesUtil, and it will be removed when the PR for portal-search-test-util is merged.
Thanks for checking!

### CODE REVIEW

**Comment by jeffwu0724** [comment]
ci:test:sf

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:search

**Comment by Kvale1733** [comment]
@jeffwu0724 No regressions.

File: modules/dxp/apps/search-experiences/search-experiences-web/src/main/java/com/liferay/search/experiences/web/internal/configuration/admin/display/SemanticSearchConfigurationFormRenderer.java [hunk 1]
```diff
@@ -249,17 +246,17 @@ private Map<String, String> _getAvailableTextTruncationStrategies(
 	}
 
 	private Map<String, String> _sortByValue(Map<String, String> map) {
-		Set<Map.Entry<String, String>> entrySet = map.entrySet();
+		Map<String, String> sortedMap = new LinkedHashMap<>();
```
Comment (dantewang) [inline modules/dxp/apps/search-experiences/search-experiences-web/src/main/java/com/liferay/search/experiences/web/internal/configuration/admin/display/SemanticSearchConfigurationFormRenderer.java:249]: @jeffwu0724 Please rename it to sortedValues.

**Pull request closed without merging**